store/views.py

```diff
@@ -77,6 +95,36 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    if request.method == "POST":
+        try:
+            bid = request.POST['bid']
+            book = BookCopy.objects.get(pk=bid)
+            book.status = True
+            book.borrower = None
+            book.borrow_date = None
+            book.save()
+            return JsonResponse( {"message":"Book successfully returned."} )
+        except:
+            return JsonResponse( {"message":"No 'bid' found"} )
+    else:
+        return JsonResponse( {"message":"Bad Request"} )
+
+@csrf_exempt
+@login_required
+def rateBookview(request, bid):
+    if request.method == "POST":
+        book = Book.objects.get(pk=bid)
```

Comment:
This may also fail with invalid book ID given in POST request, and would lead to server error. Expected behavior is to inform user with Not found (404) error.